digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -309,6 +309,12 @@ public TDJobOperator runJob(TaskState state, String key, DurationInterval pollIn
             }
             throw errorPollingException(state, key, jobState, retryInterval);
         }
+        catch (TaskExecutionException ex) {
+            if (ex.getMessage().contains("HTTP request execution failed with code 401")) {
+                updateApikey(secrets);
+            }
+            throw Throwables.propagate(ex);
```

Comment:
TaskExecutionException is RuntimeException. You don't need Throwables.propagate. Just `throw ex`.